docs/WindowsCrossCompile.md

```diff
@@ -0,0 +1,44 @@
+# Cross-compiling Swift for Windows with `clang`.
```

Comment:
Consistency: Sentence ending in period or title case heading without period? It seems the latter choice fits with the rest of the document.